tasks/rh-sign-image/rh-sign-image.yaml

```diff
@@ -92,7 +92,12 @@ spec:
             if [ $(jq -r '.mediaType' <<< $RAW_OUTPUT) == "application/vnd.oci.image.manifest.v1+json" ] ; then
               manifest_digests="${referenceContainerImage#*@}"
             else
+              manifest_list_digest="${referenceContainerImage#*@}"
               manifest_digests=$(jq -r '.manifests[].digest' <<< $RAW_OUTPUT)
+              #
+              # add manifest list digest to list of digests to sign
+              #
+              manifest_digests="$manifest_digests $manifest_list_digest"
             fi
```

Comment:
```suggestion
            # Always sign the top level sha
            manifest_digests="${referenceContainerImage#*@}"
            # For multi arch, also sign all the manifests inside
            if [ $(jq -r '.mediaType' <<< $RAW_OUTPUT) != "application/vnd.oci.image.manifest.v1+json" ] ; then
              nested_digests=$(jq -r '.manifests[].digest' <<< $RAW_OUTPUT)
              manifest_digests="$manifest_digests $nested_digests"
            fi
```

Reply:
In my proposal the order is different - the manifest list is first. So either you'd have to change the test, or you could also change the assignment. I think it's nice to start with the top level sha, so changing the test is preferable.

Reply:
I like your proposal!

Reply:
Cool!